cmd/evtest/main.go

```diff
@@ -49,6 +78,13 @@ func main() {
 		return
 	}
 
+	if err := d.Grab(); err != nil {
+		fmt.Printf("Cannot grab device: %s", err)
+		return
+	} else if err := d.Ungrab(); err != nil {
+		fmt.Printf("Cannot ungrab device: %s", err)
+	}
```

Comment:
What's the purpose of this?

Reply:
is there anything left besides removal of this section? I don't see any other blockers.

Reply:
The purpose of the `evtest` is also to debug intput devices in a system, and for that, an implicit `Grab`/`Ungrab` is rather obstructive. You could make it optional through a command line argument tho?

Reply:
Besides, an `else` is not needed when the preceding `if` block ends with a `return` statement.